gsl.copy_properties.sh

```diff
@@ -72,9 +72,9 @@ function generate_artifacts(path_prefix)
         echo(" Evaluating repository: $(_repository.name)")
         for _repository->install.build as _build where\
             defined(_build.repository) & starts_with(_build.repository, "libbitcoin")
-            emit_import_copy(_repository, my.path_prefix, _build.repository)
+            emit_import_copy(_build.repository, my.path_prefix, _build.repository)
```

Comment:
_repository seems to be a node, _build.repository is a attribute value. We try to extract data from the node inside "emit_import_copyt" and it fails.

Reply:
Looks like my generation was succeeding only due to residual files.